frontend/src/metabase/ui/components/buttons/ActionIcon/ActionIcon.styled.tsx

```diff
@@ -0,0 +1,18 @@
+import type { MantineThemeOverride } from "@mantine/core";
+
+export const getActionIconOverrides =
```

Comment:
Used for close button of the modal, not exposed on its own yet.

Reply:
@ranquild isn't `Modal` using the `CloseButton`?
https://mantine.dev/core/close-button/

Reply:
https://github.com/metabase/metabase/blob/master/frontend/src/metabase/querying/components/FilterModal/FilterModal.tsx#L105